release-notes/2022-11/README.md

```diff
@@ -0,0 +1,75 @@
+# SystemLink Enterprise 2022-11 Release Notes
+
+The 2022-11 release bundle for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. Work with your account representative to obtain credentials to access these artifacts.
+
+## New Feature
+
+- OpenAPI documentation for the Products service is now included and available at `api-hostname/niapis`.
```

Comment:
@prestwick We don't ship the /niapis route with SLE currently. That's SaaS only.

Reply:
Good catch. I've removed this.